frontend/src/pages/projects/screens/projects/ProjectTableRow.tsx

```diff
@@ -30,15 +39,19 @@ const ProjectTableRow: React.FC<ProjectTableRowProps> = ({
       <Td dataLabel="Name">
         <Flex spaceItems={{ default: 'spaceItemsXs' }} alignItems={{ default: 'alignItemsCenter' }}>
           {project.metadata.labels?.[KnownLabels.DASHBOARD_RESOURCE] && (
-            <Tooltip content="Data Science">
-              <Label isCompact color="green">
-                DS
-              </Label>
-            </Tooltip>
+            <FlexItem className="odh-projects__label">
```

Comment:
We should try to avoid adding customized class. Maybe use inline style instead if needed?
```suggestion
            <FlexItem style={{ display: 'flex' }}>
```